ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/SummaryCards/CvesByStatusSummaryCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
These summary card components are _barely_ reusable, if at all. Do you think it is cleaner to keep them in separate files like this, or just inline them in the main `ImageSingleVulnerabilities` component?

Reply:
i think keeping them separate makes sense since the `ImageSingleVulnerabilities` component is already 200+ lines long.